modules/v2x/v2x_proxy/obu_interface/grpc_interface/grpc_server.cc

```diff
@@ -80,49 +96,70 @@ grpc::Status GrpcServerImpl::SendV2xTrafficLight(
   }
   {
     std::lock_guard<std::mutex> guard(traffic_light_mutex_);
-    latest_trafficlight_.CopyFrom(*request);
-  }
-  if (first_recv_flag_) {
-    auto msg = std::make_shared<StatusResponse>();
-    msg->set_status(true);
-    if (first_flag_writer_->Write(msg)) {
-      AINFO << "grpc sync flag send success";
-      first_recv_flag_ = false;
-    } else {
-      AERROR << "grpc sync flag send failed";
+    latest_traffic_light_.CopyFrom(*request);
+    auto obu_header = latest_traffic_light_.mutable_header();
+    if (obu_header->has_timestamp_sec()) {
+      ADEBUG << "OBU Timestamp: " << std::setiosflags(std::ios::fixed)
+             << std::setprecision(11) << obu_header->timestamp_sec();
+      ADEBUG << "SVB Timestamp: " << std::setiosflags(std::ios::fixed)
+             << std::setprecision(11)
+             << ::apollo::cyber::Time::Now().ToSecond();
+      obu_header->set_timestamp_sec(::apollo::cyber::Time::Now().ToSecond());
```

Comment:
There seems to be a format_timestamp in modules/common. Please reuse that.
And Don't call now() two times.